patches/chrome-browser-resources-settings-autofill_page-passwords_section.html.patch

```diff
@@ -1,11 +1,12 @@
 diff --git a/chrome/browser/resources/settings/autofill_page/passwords_section.html b/chrome/browser/resources/settings/autofill_page/passwords_section.html
-index 75ac76d97ae9864b7b619bec9407d3b14233181e..a4bb488201b3ac9d43134b242d5ea702f995b6a9 100644
+index 2c62d1493d1d6b2257231301c580676e6f4e3693..c308ed981596aec8277cabe60204a7b61ac6fee4 100644
 --- a/chrome/browser/resources/settings/autofill_page/passwords_section.html
 +++ b/chrome/browser/resources/settings/autofill_page/passwords_section.html
-@@ -143,11 +143,15 @@
-         </cr-icon-button>
-       </div>
-     </template>
+@@ -141,11 +141,16 @@
+           aria-label="$i18n{checkPasswords}">
+       </cr-icon-button>
+     </div>
++    </template>
```

Comment:
this doesn't seem right, am I reading this diff wrong or are we adding a `</template>` tag?

Reply:
Fixed in 72791a9df18e1782bad4c0db507768f1d2c59012